Test/Python/CreatePipeline.py

```diff
@@ -0,0 +1,106 @@
+
+    # Rotate the Crystal reference frame
+    filter = orientationanalysis.RotateEulerRefFrame()
+    filter.RotationAxis = simpl.FloatVec3([0.0, 0.0, 1.0])
+    filter.RotationAngle = 90.0
+    filter.CellEulerAnglesArrayPath = simpl.DataArrayPath(data_container_name, cell_attribute_matrix_name, 'EulerAngles')
+    pipeline.pushBack(filter)
+
+    # Threshold the Data by Confidence Index
+    filter = simpl.MultiThresholdObjects2()
+    thresholds = simpl.ComparisonInputsAdvanced()
+    topLevelComparisonSet = simpl.ComparisonSet()
+    comparisonValue = simpl.ComparisonValue()
+    comparisonValue.AttributeArrayName = 'Confidence Index'
+    comparisonValue.CompOperator = 1 # '>'
+    comparisonValue.CompValue =  0.1
+    topLevelComparisonSet.addComparison(comparisonValue)
+    thresholds.addInput(topLevelComparisonSet)
+    thresholds.DataContainerName = data_container_name
+    thresholds.AttributeMatrixName = cell_attribute_matrix_name
+    filter.DestinationArrayName = 'Mask'
+    filter.SelectedThresholds = thresholds
+    pipeline.pushBack(filter)
+
+    # Generate IPF Colors
+    filter = orientationanalysis.GenerateIPFColors()
+    filter.ReferenceDir = simpl.FloatVec3([0, 0, 1])
+    filter.CellPhasesArrayPath = simpl.DataArrayPath(data_container_name, cell_attribute_matrix_name, 'Phases')
+    filter.CellEulerAnglesArrayPath = simpl.DataArrayPath(data_container_name, cell_attribute_matrix_name, 'EulerAngles')
+    filter.CrystalStructuresArrayPath = simpl.DataArrayPath(data_container_name, cell_ensemble_attribute_matrix_name, 'CrystalStructures')
+    filter.UseGoodVoxels = True
+    filter.GoodVoxelsArrayPath = simpl.DataArrayPath(data_container_name, cell_attribute_matrix_name, 'Mask')
+    filter.CellIPFColorsArrayName = 'IPFColor'
+    pipeline.pushBack(filter)
+
+    # ITK Image Writer
+    image_file = "{}/Data/Output/Python_Examples/CreatePipeline/Small_IN100_1.png".format(sd.GetBuildDirectory())
+    filter = itkimageprocessing.ITKImageWriter.New()
+    filter.registerImageIOFactories()
+    filter.FileName = image_file
+    filter.ImageArrayPath = simpl.DataArrayPath(data_container_name, cell_attribute_matrix_name, 'IPFColor')
+    filter.Plane = 0
+    pipeline.pushBack(filter)
+
+    # Write out a .dream3d file which will also save a copy of the pipeline as JSON
+    output_file = "{}/Data/Output/Python_Examples/CreatePipeline/Small_IN100_1.dream3d".format(sd.GetBuildDirectory())
+    filter = simpl.DataContainerWriter.New()
+    filter.OutputFile = output_file
+    pipeline.pushBack(filter)
+
+    # We can preflight the pipeline at this point
+    pipeline.preflightPipeline()
+    print(f'Preflight error = {pipeline.ErrorCode}')
+
+    # We can now run the built up pipeline
+    dca = pipeline.run()
+    print(f'Execution error = {pipeline.ErrorCode}')
+
+if __name__ == '__main__':
+    print('Starting the Pipeline construction example...')
+    CreatePipeline()
+    print('[CreatePipeline] Complete')
```

Comment:
```suggestion
    print('[CreatePipeline] Complete')
    
```
Add newline to end of file.